Frameworks/Foundation/NSKeyValueCoding.mm

```diff
@@ -256,6 +255,24 @@ static bool tryGetArrayAdapter(id self, const char* key, id* ret) {
     return true;
 }
 
+static bool tryGetSetAdapter(id self, const char* key, id* ret) {
```

Comment:
this and similar files in non-core projects seem to be using a different convention so I followed that one